IMC_WebDev/FromDayToNight-CreatingInteractiveColorPalettesWithJavaScript/blog.md

```diff
@@ -0,0 +1,394 @@
+
+#moon {
+    position: absolute;
+    left: 350px;
+    bottom: 75px;
+    width: 115px;
+    height: 115px;
+    border-radius: 50%;
+    background: radial-gradient(#D3D3D3, #555AA6);
+}
+...
+```
+
+For our moon, we will use a new selector labelled `#moon` and change the colors in the `radial-gradient` function—the color at the center of our moon will be a light cool gray, and the color at the edges will be a dark grayish violet.
+
+At this point, we will not be able to view our moon element because we have not declared a `<div>` element for it in our HTML code. We do not need to declare this because will only be switching back and forth between these two selectors, which we will code in JavaScript later on.
+
+This is what our completed `#sun` and `#moon` elements should look like:
+
+```css
+...
+#sun {
+    position: absolute;
+    left: 350px;
+    bottom: 75px;
+    background: radial-gradient(#FFF89E, #FFDE45);
+    width: 115px;
+    height: 115px;
+    border-radius: 50%;
+}
+
+#moon {
+    position: absolute;
+    left: 350px;
+    bottom: 75px;
+    width: 115px;
+    height: 115px;
+    border-radius: 50%;
+    background: radial-gradient(#D3D3D3, #555AA6);
+}
+...
+```
+
+### CSS Code: Day and Night Buttons
+
+To start stylizing our buttons' properties, we will be adding a `button` selector into our code:
+
+```css
+...
+<style TYPE="text/css">
+    ...
+    button {
+
+    }
+</style>
+...
+```
+
+This selector accesses all `<button>` elements declared in the HTML code, which are our `day` and `night` buttons.
+
+First, we will adjust the color and position of our buttons with the following attributes:
+
+```css
+...
+button {
+	position: relative;
+    top: -40px;
+    right: -360px;
+    background-color: #FFFFFF;
+}
+...
+```
+
+By setting the `position` attribute to `relative`, this will allow us to adjust the location of the buttons from its "normal" position, which is the center of the webpage, using the `top` and `right` attributes—the values that we inputted will move buttons to the area below our sun. We will also set the buttons' color to white using the `background-color` attribute.
+
+Then, we will add a border to our buttons by adding the following attributes:
+
+```css
+...
+button {
+	...
+    border-style: solid;
+    border-width: 2px;
+    border-color: #FF8080;
+    border-radius: 10px;
+}
+...
+```
+
+For all the border properties to show up, the `border-style` attribute first must be set. Above, we set our buttons' borders to `solid` that will result in a solid colored border. Feel free to experiment with different kinds of borders from this [CSS Border Styles list](https://www.w3schools.com/css/css_border.asp). As for the additional border attributes, `border-width` and`border-color` adjust the border's thickness and color, respectively, and `border-radius` adjusts the roundness of the corners of the border—this particular property allows up to four values, each value accessing each corner of the element.
```

Comment:
"the border—this particular property" — for these dashes, you need a space on either side — like this! (please  change this for all the big dashes that act like this one. You don't use spaces when you are using these **smaller-sized** dashes to connect to words)